mappings/net/minecraft/entity/projectile/ProjectileEntity.mapping

```diff
@@ -4,12 +4,20 @@ CLASS net/minecraft/class_1676 net/minecraft/entity/projectile/ProjectileEntity
 	FIELD field_28646 shot Z
 	FIELD field_33399 owner Lnet/minecraft/class_1297;
 	METHOD method_24919 setProperties (Lnet/minecraft/class_1297;FFFFF)V
```

Comment:
Imo this should be setting velocity as well. This just takes account of the shooter's velocity so the input velocity is relative.